eng/Versions.props

```diff
@@ -131,9 +131,11 @@
     <OmniSharpExtensionsLanguageServerPackageVersion>0.14.2</OmniSharpExtensionsLanguageServerPackageVersion>
     <OmniSharpMSBuildPackageVersion>1.33.0</OmniSharpMSBuildPackageVersion>
     <SystemPrivateUriPackageVersion>4.3.2</SystemPrivateUriPackageVersion>
+    <SystemCompositionPackageVersion>1.0.31.0</SystemCompositionPackageVersion>
```

Comment:
Don't need two of em.

Reply:
Thanks, that was a bad merge on my part. Removed the duplicate.